airflow/jobs/triggerer_job_runner.py

```diff
@@ -252,14 +252,12 @@ def __init__(
         job: Job | JobPydantic,
         capacity=None,
     ):
-        super().__init__()
+        super().__init__(job)
         if job.job_type and job.job_type != self.job_type:
```

Comment:
Move this check in to the base class too?

Reply:
Sorry forgot about that I wasn't fully focused, was fighting to get Airflow running locally on ARM macbook for the first time it's not really easy haha